app/views/newflow/educator_signup/educator_signup_form.html.erb

```diff
@@ -121,6 +121,17 @@
         </div>
 
         <div class="content control-group checkboxes-section">
+
+          <% if is_BRI_book_adopter? %>
```

Comment:
This block is in 3 different forms. Should it be a fragment so it only exists in one place?

Reply:
Great suggestion, thanks! Done.